lambdas/postcode-lookup/src/test/java/uk/gov/di/ipv/cri/address/api/handler/PostcodeLookupHanderTest.java

```diff
@@ -37,10 +37,7 @@
 import static org.mockito.ArgumentMatchers.isNotNull;
 import static org.mockito.ArgumentMatchers.isNull;
 import static org.mockito.BDDMockito.willThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
We try to avoid * imports

Reply:
This happened automatically by Intellij, I think I reset my IDE back to default so it did this. Updated now to be explicit.